srcpkgs/python3-ipyparallel/template

```diff
@@ -7,15 +7,24 @@ hostmakedepends="hatchling"
 depends="python3-entrypoints python3-decorator python3-pyzmq python3-traitlets
  python3-ipython python3-jupyter_client python3-ipython_ipykernel
  python3-tornado python3-psutil python3-dateutil python3-tqdm"
+checkdepends="$depends python3-pytest python3-pytest-asyncio
+ python3-testpath python3-matplotlib python3-joblib python3-numpy
+ python3-pandas"
 short_desc="Interactive parallel computing in Python"
 maintainer="Andrew J. Hesford <ajh@sideband.org>"
 license="BSD-3-Clause"
 homepage="https://ipyparallel.readthedocs.io/"
 changelog="https://ipyparallel.readthedocs.io/en/latest/changelog.html"
 distfiles="${PYPI_SITE}/i/ipyparallel/ipyparallel-${version}.tar.gz"
 checksum=a39aa5ef9560170bf0e9afedca9ff045e1b9c1832c49303377edcc91cea9fb77
-# Too many tests fail with a "local" cluster type
-make_check=no
+
+# Seems broken in musl
+[[ "$XBPS_TARGET_MACHINE" = *-musl ]] && make_check=no
+
+pre_check() {
+	# having HOME=/tmp confuses a few tests
+	HOME=/home
+}
```

Comment:
This should just be `make_check_pre="env HOME=/home"`, both because it is simpler and will avoid setting `$HOME` in a way that could influence later build stages.